<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergeDialog</class>
 <widget class="QDialog" name="MergeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Merge Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="grpLocal">
       <property name="title">
        <string>Local Actions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpCloud">
       <property name="title">
        <string>Cloud Actions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpMerged">
       <property name="title">
        <string>Merge Result</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkShowResult">
     <property name="text">
      <string>Show merge result</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnDeny">
       <property name="text">
        <string>Deny</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/assets/images/button-icons/cancel.png</normaloff>:/assets/images/button-icons/cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConfirm">
       <property name="text">
        <string>Confirm</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/assets/images/button-icons/ok.png</normaloff>:/assets/images/button-icons/ok.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
